udata_front/theme/gouvfr/assets/js/components/ProducerSelector/ProducerSelector.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div v-if="isAdmin()">
+    <MultiSelect
```

Comment:
I think the `initialOptions` props should be provided with a list of the user organizations